stories/Notification/Notifications.stories.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs } from '@storybook/addon-knobs/react'
+import { withA11y } from '@storybook/addon-a11y'
+import { withInfo } from '@storybook/addon-info'
+
+import { Container, Row, Col, Notifications } from '../../src'
+import notifJson from './notifications.json'
+
+storiesOf('Componenti/Notifications', module)
+  .addDecorator(withA11y)
+  .addDecorator(withKnobs)
+  .add(
+    'Esempi',
+    withInfo({
+      // text: Esempi
+    })(() => {
+      const { notificationId, NotificationManager } = Notifications
+      const notifRef = React.createRef()
+      const showNotification = (e, info) => {
+        notifRef.current.show(e, info)
+      }
```

Comment:
The API looks a bit fidly to me:
* get an id generator and the manager
* get the element ref to access the emitter
* pass the stuff to the emitter...